addons/sale/models/sale.py

```diff
@@ -1620,6 +1620,61 @@ def product_id_change(self):
 
         return result
 
+    def _get_computed_price_unit(self, product=None):
+        ''' Helper to get the default price unit based on the product by taking care of the taxes
```

Comment:
This requires at least a TODO in comment explaining that this code mostly a copy/pasted version of ``_fix_tax_included_price_company`` that had to be defined apart on a stable release, but that should be merged into ``_fix_tax_included_price_company`` in master. 

Otherwise, we'll probably loose the origin of this, and forgot to clean the final version.

Reply:
In fact, the `_fix_tax_included_price_company` is wrong and leads to unconsistencies regarding the taxes computation, the python tax management etc. I started a task to unify the computation for all business models (see https://runbot.odoo.com/runbot/bundle/master-generic-taxes-comp-las-96494).

Instead of copy pasting all this behavior from one model to another, I suggest an helper method somewhere doing the same behavior like what I did for the taxes computation.

Also, there is some test in accounting you could reuse for sale order:
https://github.com/odoo/odoo/blob/13.0/addons/account/tests/test_account_move_out_invoice.py#L192
https://github.com/odoo/odoo/blob/13.0/addons/account/tests/test_account_move_out_invoice.py#L335